modules/usage_alarms/variables.tf

```
@@ -33,3 +33,9 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "dp_services" {
```

What does `dp` mean here?

```suggestion
variable "dp_services" {
```

---

refering to GDP